app/(root)/page.tsx

```
@@ -1,7 +1,30 @@
+import { tryCatch } from "@/lib/utils/tryCatch";
+import Link from "next/link";
+
 export default async function Home() {
+  const { data, error } = await tryCatch(
+    notion.databases.query({
+      database_id: NOTION_DATABASE_ID,
+      auth: NOTION_SECRET,
+    })
+  );
+
+  error ? console.error(error) : console.log(data);
```

ESLint

[error] Expected an assignment or function call and instead saw an expression.

(@typescript-eslint/no-unused-expressions)